index.js

```diff
@@ -1,92 +1,114 @@
-var fs    = require('fs'),
+'use strict';
+
+const fs  = require('fs'),
     path  = require('path'),
     spawn = require('child_process').spawn,
+    async = require('async'),
```

Comment:
I'm a big fan of bluebird. However, the module at <1.0.0 uses callbacks by convention. If we move towards promises I think we aught to completely adopt it privately and publicly. We can make that a todo for getting to 1.0